search.html

```diff
@@ -0,0 +1,21 @@
+---
+layout: default
+title: Search
+---
+<br><br><center>
+<form id="cloudsh">
+    <input type="text" id="q"/>
+    <button>Search</button>
+</form>
+</center>
+<h2>Results</h2>
+<div id="results"></div>
+
+<script src="https://app.cloudsh.com/js/cloudsh.js"></script>
+<script>
+    var CshConf = {
+        token: "eyJhbGciOiJIUzI1NiJ9.eyJ1c2VyX2lkIjoiMWVhZGI4NWItMDgwNS00MDRkLWJhYmMtOGQ5Mjc5OGYzNDliIiwiaW5kZXhfaWQiOiIyMzk3Yjk4Yy1lMTk3LTRjMGQtOTlkYS04NGIxOWEzMjk1ZjEiLCJpbmRleF91cmwiOiJodHRwOi8vcGNsdWIuaW4ifQ.49yI9ac98bXkeoIzXxW_LRA_tTJgHeAErr_aVgu1MQg",
```

Comment:
Are these tokens for public use?
Private tokens should not be placed in codebase.